webpack/move_to_foreman/components/common/ModalProgressBar/ModalProgressBar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, ProgressBar } from 'patternfly-react';
+
+const ModalProgressBar = (props) => {
+  const { show, container, task } = props;
+  let modalTitle = null;
+  let progress = 0;
+
+  if (task) {
+    modalTitle = task.humanized.action;
+    progress = Math.round(task.progress * 100);
```

Comment:
Could probably factor this out and make a task progress component.

Reply:
why not factor it out pre-emptively? Seems like its less likely to get done later rather than doing it now